autoscheduler/frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -260,7 +260,8 @@ const Schedule: React.FC<RouteComponentProps> = () => {
         availability={availability}
         firstHour={FIRST_HOUR}
         lastHour={LAST_HOUR}
-        key={formatTime(availability.startTimeHours, availability.startTimeMinutes, true)}
+        key={`${formatTime(availability.startTimeHours, availability.startTimeMinutes, true)}
+        - ${formatTime(availability.endTimeHours, availability.endTimeMinutes)}`}
```

Comment:
Although this has no effect on functionality, both of the `formatTime` functions should use 24 hour time for consistency

Reply:
oops my bad